src/templates/fr/2019/base_chapter.html

```diff
@@ -0,0 +1,146 @@
+{% extends "fr/2019/base.html" %}
+
+{% block title %}{{ metadata.get('title') }} | 2019 | Le Web Almanac par HTTP Archive{% endblock %}
+
+{% block styles %}
+{{ super() }}
+<link rel="stylesheet" href="/static/css/page.css">
+{% endblock %}
+
+{% block scripts %}
+{{ super() }}
+<script nonce="{{ csp_nonce() }}">
+  
+var indexBox = document.querySelector('.index-box');
+var indexBoxTitle = document.querySelector('.index .header');
+ 
+indexBoxTitle.addEventListener('click', function(e) {
+  indexBox.classList.toggle('show');
+});
+
+document.addEventListener("keypress", function onPress(event) {
+ 	if (event.key === 'p' || event.key === 'P' || event.key === ',' || event.key === '<') {
+ 		var previous = document.getElementById('previous-chapter');
+ 		if (previous) {
+ 		  previous.click();
+ 		}
+ 	}
+ 	if (event.key === 'n' || event.key === 'N' || event.key === '.' || event.key === '>') {
+ 		var next = document.getElementById('next-chapter');
+ 		if (next) {
+ 		  next.click();
+ 		}
+ 	}
+});
+
+window.discussion_url="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}.json";
+</script>
+
+<script src='/static/js/chapter.js' defer></script>
+
+{% endblock %}
+
+{# Calls to action for readers who want to engage more with this chapter. #}
+{% macro render_actions() %}
+  <div class="discuss">
+    <a class="alt btn" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
+      <img src="/static/images/discuss.png" alt="Discuss this chapter" height="24" width="24" loading="lazy" /><span id="num_comments"></span><span data-translation id="comment-singular" style="display:none;">commentaire</span><span data-translation id="comment-plural">commentaires</span>
```

Comment:
Can we remove the in-line style? In case we ever remove this from CSP. Also you only have it on one and not the other.

```suggestion
      <img src="/static/images/discuss.png" alt="Discuss this chapter" height="24" width="24" loading="lazy" /><span id="num_comments"></span><span data-translation id="comment-singular">commentaire</span><span data-translation id="comment-plural">commentaires</span>
```